### DISCUSSION TITLE
Language detection detects Smarty instead of YAML in Helm Charts repo

### DISCUSSION THREAD

**Discussion by okaufmann** [community/community#71689] [Repositories]
### Select Topic Area

Bug

### Body

Hi

In my [helm charts repository](https://github.com/okaufmann/helm-charts), GitHub detects the main Language as "Smarty". The File endings are `.yaml` and also the contents are mainly YAML.

<img width="324" alt="image" src="https://github.com/community/community/assets/4414498/b4d9d33e-a034-4500-bf41-82c5a39c7f18">

If this is not a Bug: can I somehow define what language, GitHub should detect the files as?

**Comment by vincentscode** [comment]
This is likely related to this (and similar) never resolved issue in github-linguist: https://github.com/github-linguist/linguist/issues/6196. It causes _helpers.tpl to be identified as Smarty.
